src/Auth0.php

```diff
@@ -93,11 +94,16 @@ public function login(
         ?string $redirectUrl = null,
         ?array $params = null
     ): string {
+        if (! $this->configuration()->usingStatefulness()) {
+            throw ConfigurationException::requiresStatefulness('Auth0->login()');
+        }
+
         $this->deferStateSaving();
 
+        $store = $this->getTransientStore();
         $params = $params ?? [];
-        $state = $params['state'] ?? $this->getTransientStore()->issue('state');
-        $params['nonce'] = $params['nonce'] ?? $this->getTransientStore()->issue('nonce');
+        $state = $params['state'] ?? $store?->issue('state') ?? uniqid();
```

Comment:
Null-safe operators are a new language feature in PHP 8.0+ which offers cleaner syntax than null coalescing, as that does not work on method calls. I'll be transitioning the SDK to use these as time goes on.